domain/src/main/java/camp/nextstep/edu/domain/ExpressionHandler.kt

```diff
@@ -0,0 +1,16 @@
+package camp.nextstep.edu.domain
+
+class ExpressionHandler {
+
+    private var expression = ""
+
+    fun getExpression() = expression
```

Comment:
코틀린의 멤버변수는 프로퍼티의 특징을 가지고 있습니다.
따라서 getter 함수를 새로 만들기보다는 기본적으로 접근 가능하게 하는 것이 더욱 좋아요!

```suggestion
    var expression = ""
        private set
```

위와 같은 형태로 변경해보는 것은 어떨까요?!

Reply:
앗..!! 그렇네요..!! 감사합니다ㅎㅎ